XamlControlsGallery/App.xaml

```diff
@@ -122,6 +122,7 @@
                 <Setter Property="HorizontalContentAlignment" Value="Stretch" />
             </Style>
 
+            <x:Double x:Key="TeachingTipMinWidth">50</x:Double>
```

Comment:
@StephenLPeters, is there a better way to override this than having this resource in the App.xaml file? I tried setting this in TeachingTip.ResourceDictionary however that did not work (or I did something wrong there). Maybe it's worth introducing properties so one can override those properties on a per control basis (unless lightweight styling using the controls ResourceDictionary works, then not).

Reply:
This seems to be the best way to do it for now (https://stackoverflow.com/questions/59542235/unable-to-set-width-height-property-of-teaching-tip-in-uwp)

Reply:
Guess it's just one of those popup layer problems that we can't get around for now, thanks for checking @gregwoo-microsoft! Maybe this is worth adding a property on the teachingtip, but that's a discussion for a different time.

Reply:
Yes I agree. I'll approve the changes then :)